endpoints/openrtb2/sample-requests/valid-whole/supplementary/aliased-buyeruids.json

```diff
@@ -1,5 +1,6 @@
 {
   "description": "Amp request where root.user.ext.prebid.buyeruids field makes use of alias defined in root.ext.prebid.aliases",
+  "query": "tag_id=101",
```

Comment:
Can we move these AMP specific tests into a new directory or rename one of the existing folders in the path so we know these are AMP tests?

Reply:
Done. `sample-requests/valid-whole/supplementary/aliases.json` was duplicated into: `sample-requests/amp/valid-supplementary/aliases.json`. The latter comes with the `"query"` field, the former, doesn't.